### ISSUE TITLE
errors when using Fn::Transform inside mappings

### ISSUE THREAD

**Issue by victoru** [aws-cloudformation/cfn-python-lint#1856]
*cfn-lint version: cfn-lint 0.44.1*

When using `Fn::Transform` inside a mapping like this
```yaml
Mappings:
  SubMapOne:
    'Fn::Transform':
      Name: 'AWS::Include'
      Parameters:
        Location: s3://foo.net/bar.yml
```

I get this error
```
E7001: Mapping SubMapOne has invalid property at Parameters
```

But this usage is valid since I was able to create the stack successfully...

If I do this:
```yaml
Mappings:
  'Fn::Transform':
    Name: 'AWS::Include'
    Parameters:
      Location: >-
        Location: s3://foo.net/bar.yml
```

I get this error

```
E7002: Mapping Fn::Transform has invalid name.  Name has to be alphanumeric.
```

I've not tested this one yet, but [the AWS docs state](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/create-reusable-transform-function-snippets-and-add-to-your-template-with-aws-include-transform.html#aws-include-transform-usage):
> You can use the AWS::Include transform anywhere within the AWS CloudFormation template except in the template parameters section or the template version field. > For example, you can use AWS::Include in the mappings section. 

So it seems that cfn-lint is semantically ignoring `Fn::Transform` when in a mapping.

**Comment by PatMyron** [comment]
tracking in https://github.com/aws-cloudformation/cfn-python-lint/issues/476
